cli/internal/nodes/packagetask.go

```diff
@@ -33,10 +33,13 @@ func (pt *PackageTask) OutputPrefix(isSinglePackage bool) string {
 // of this task
 func (pt *PackageTask) HashableOutputs() fs.TaskOutputs {
 	inclusionOutputs := []string{fmt.Sprintf(".turbo/turbo-%v.log", pt.Task)}
+
+	var tdOutputs []string
 	inclusionOutputs = append(inclusionOutputs, pt.TaskDefinition.Outputs.Inclusions...)
+	tdOutputs = pt.TaskDefinition.Outputs.Exclusions
```

Comment:
If the declaration being separate from initialization is required a comment explaining why would be helpful as otherwise I would assume the following would be equivalent: 
```suggestion
	inclusionOutputs = append(inclusionOutputs, pt.TaskDefinition.Outputs.Inclusions...)
	tdOutputs := pt.TaskDefinition.Outputs.Exclusions
```

Reply:
vestige from before, where this was conditionally assigned if `TaskDefinition.Outputs != nil`